### DISCUSSION TITLE
「電子鼻」可辨識威士忌真假

### DISCUSSION THREAD

**Discussion by yi-1111** [1101-GS4719-C/Python-Community-GS4719-C#568] [科技新知]
近期澳洲雪梨工程大學最新一期的研究成果，研發出可以媲美品酒專家的電子鼻，來檢測威士忌的年份甚至有高達 95% 的準確度。
根據大學副教授 Steven Su 稱，這種電子鼻最初是用以檢測黑市販售的非法動物製品，如犀牛角，經改裝後才轉向測量食物品質等，而團隊在把電子鼻調整後，配上一個收集威士忌樣本的瓶子，找出最重要的氣味特徵，再由機器學習演算法分析後辨識出威士忌品牌、地區和風格。

對於愛喝酒的捧油算是一大福音吧
https://buzzorange.com/techorange/2022/04/11/nos-e-whisky/

**Comment by HuanYuuuuuu** [comment]
感覺很厲害，不知道是不是可以利用電子鼻來進一步判斷該未知物的成分等等

**Comment by d26488399** [comment]
好厲害，這樣可以更容易抓到那些標示不實的不肖廠商

**Comment by Yuhsin000** [comment]
好像小時候在下課花路米有看過這個技術，不過那時候看到的是很大一台的機器

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，好神奇的機器。

**Comment by yenwei12138** [comment]
這技術發展成熟的話，應該更適合使用在緝毒或非法添加物的檢驗上吧，只用在酒上感覺偏大材小用

**Comment by lx3211** [comment]
電子鼻拿來取代緝毒犬也是很不錯的方向

**Comment by handsomeniko** [comment]
確實厲害ㄟ

**Comment by cordydeng** [comment]
文章中有提到除了威士忌以外還有在訓練判斷其他酒的品種，如果真的可以成功發展，想必可以遏制假酒市場及保障愛酒人士的權益

**Comment by Sazmafuji** [comment]
這應該可以用在許多地方吧

**Comment by stan890425** [comment]
威士忌有多種香味，能變出來真的很猛

**Comment by sys511613** [comment]
這個東西還可以用在超多地方，而且文章中有提到透過AI的力量已經可以跟最先進的GC×GC-ToFMS表現相當，真的很厲害

**Comment by jcpan821** [comment]
現在連警犬也要失業了

**Comment by alexthebigpp** [comment]
感覺也可以用來偵測其他東西，像毒品之類的

**Comment by Ben6uo** [comment]
謝謝分享 這真的解決了不少問題

**Comment by etin2468** [comment]
這可能在犯罪偵查有很好的應用

**Comment by bloodtrail0511** [comment]
太酷了吧!!

**Comment by Lynn12378** [comment]
同意樓上，若拿來緝毒感覺會很棒

**Comment by Jimmy911218** [comment]
感覺可以拿來其他的用途更能造福世界

**Comment by Hchejxnc** [comment]
用來緝毒真的感覺很棒!

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
這項技術的普及應該能運用在更廣泛的地方 不單單只用在品酒

**Comment by Pietra1226** [comment]
感覺也可以用在海關或是犯案現場，就不用都攜帶一隻緝毒犬了

**Comment by Zoe06060606** [comment]
連品酒師都要被取代了，這樣感覺越來越多職業要被機器取代了
雖然也產生許多新職業就是了

**Comment by Jasper525** [comment]
這對因為確診失去嗅覺的人是一大福音呢~

**Comment by bonny123865** [comment]
這有點太強啦!
看到這篇文章之前根本不知道有電子鼻這種東西欸!

**Comment by Kotori0501** [comment]
感謝分享
雖然我不喝酒，因此也不太懂每一種酒的差別在哪
如果有這技術，就可以分辨酒的種類，我覺得不錯，但如果可以好好的有效利用在其他地方（例如：毒品），就可以不用警犬之類的動物來盤查了

**Comment by evanchou1204** [comment]
聽起來就跟特異功能一樣，好神奇，讓我想到傳說中的好鼻師。

**Comment by ruthchen1020** [comment]
感測器就是人類的五官pro，我們根本打不贏。

**Comment by Zoe06060606** [comment]
這東西真的好例害 能辯別出各式味道 應該還可以用到各大領域

**Comment by CPCpyt** [comment]
我覺得可以幫助人們有效鑑定一些東西的真偽，
測驗其是否有如標榜上的一樣，辦免受騙。

**Comment by swerfg03joe** [comment]
機器學習用在分析，如果要舉簡單的例子來比喻他很像多項式方程式，不同的酒其成分有不同的比例，而AI就是將這些成分比例數據送進某種數學模型，如randomforest,然後返回一個最接近的答案

**Comment by BlahFish1006** [comment]
好厲害! 如果能將這個技術應用在分析中藥材的品質優劣，就可以有效防範不肖商人，
避免買到假的藥材，才不會越補越大洞......

**Comment by Tuoresa** [comment]
感覺超級厲害！好像科幻電影裡看見或聽見一個東西立刻在眼前條列出它的各項特質一樣
但感覺這樣就即將取代品酒師這麼一個具有魅力的職業了欸
用語言形容相比用數據形容還是更令人心動一點哈哈

**Comment by Guagua1234** [comment]
身為一個愛喝酒的人，
從250到10000元的威士忌都有喝過，
酒精的好與壞真的是分得出來的，
從一打開時的香氣到喝下去的香味停留在口腔的時間等就得以知道是否為高級的威士忌，
雖然這個產品很方便，但這樣就少了些喝酒的樂趣。
有些時候比起機器的溝通，還是透過自己慢慢摸索或是和朋友交流那些資訊來的有溫度。
畢竟一起喝酒就是為了活絡氣氛吧

**Comment by a45320614** [comment]
不知道這種技術能不能應用在安檢甚至搜救工作，利用電子鼻來分析環境中的氣味，減輕緝毒犬及搜救犬的工作負荷。

**Comment by harrylee1971** [comment]
該不會，蒐集特徵又是線代的eigenvector那些咚咚了吧，好可怕

**Comment by Hahahahbla** [comment]
對於品酒愛好者來說，這真是一項令人振奮的技術突破！這意味著我們能夠更加準確地鑑定威士忌的品牌和年份，而不必依賴專業的品酒師。這將為我們帶來更深入的品酒體驗和更多的樂趣。期待將來看到這項技術在品酒界的應用！

**Comment by yyujimmy** [comment]
這項來自澳洲雪梨工程大學的研究成果確實讓人感到振奮！這種電子鼻的應用範圍非常廣泛，從檢測非法動物製品到評估食物品質，再到如今的威士忌品評，都展現了其潛力和多功能性。對於愛喝酒的人來說，這絕對是一個有趣且實用的工具，尤其對於那些對於威士忌的年份和品牌有興趣的人來說。透過這種電子鼻，他們可以更快速地辨別和評估威士忌的品質，而不需要依賴專業品酒師的評論。這也提醒我們，科技的進步不僅能夠改善我們的生活品質，還能夠為保護動物和食品安全等議題做出貢獻。

**Comment by Piyenparty** [comment]
雖然科技很強大，可是像品酒師這種職業的魅力，真的不只是「分析」而已。他們講起酒的時候那種詩意、那種把風味變成畫面的能力，是數據永遠模仿不來的。什麼「像在午後陽光下的青蘋果樹下睡著」這種描述，怎麼用科學儀器講都講不出那個感覺哈哈。

所以就算科技能輔助判斷，品酒師還是有他們獨特、無法取代的價值在。畢竟打動人心的，從來都不只是精準，而是共鳴。

**Comment by wuyu06** [comment]
這技術好酷 感覺可以用在邊境檢查 或許可以比緝毒犬更加穩定 
或是可以幫助嗅覺失靈的人重新聞到味道 應該會是一項偉大的發明

**Comment by edawdewv** [comment]
澳洲雪梨工程大學開發的這款電子鼻技術，真的是一大突破。它不僅能在非法動物製品檢測和食物品質把關上發揮作用，現在還被應用在威士忌的品評，這讓我覺得科技正逐步滲透到我們日常生活的細節裡。對於喜歡威士忌的人來說，這種裝置能幫助快速判斷酒的品質和年份，降低對專業品酒師的依賴，也讓品酒變得更大眾化。
